scripts/inner/kubestellar-kube-bind

```diff
@@ -86,21 +74,18 @@ if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
     exit 0
 fi
 
-if [[ ! -d $log_dir ]]; then
-    mkdir -p $log_dir
-fi
-
-if [ "$start_konnector" == true ]
-then
+if pgrep konnector | xargs ps e -p 2>/dev/null | grep "KB_CONSUMER=$space_name\s" &> /dev/null; then
```

Comment:
What if KB_CONSUMER is the last variable listed on the line?

```
mspreitz@mjs13 kubestellar % grep 'foo\s'
bar
foo
doo
```

Reply:
I saw that this `KB_CONSUMER` always be the 1st in my tests, and it can't be the last because there are `PATH`, `PWD` etc.

Reply:
On my Mac the environment variables are not listed in alphabetical order.
I think it would be best to not rely on assumptions about how `ps` orders that listing.

Reply:
Addressed in latest push.